### DISCUSSION TITLE
学游泳 | /posts/learn-swim/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#583] [Announcements]
https://yuanfan.rbind.io/posts/learn-swim/

**Comment by residualsun1** [comment]
我小学五六年级的暑假就天天到其他小区的游泳池🏊‍，刚开始啥也不会，就在中水区（我们那中水1.5米，深水1.7，但实际可能没那么深）玩，后来水性熟悉了，就摸到深水区边缘徘徊，看看那些大人怎么蛙泳，就照葫芦画瓢笨手笨脚学起来。起头也喝过水，特别是要想办法浮起来的时候。去得多了教练也认识我了，有时候会对我的动作说两句，我就有意识地按照他说的做。就这样去了一个暑假，后来我继续不断在深水区边缘试探，最后慢慢就能自己游了。

可能动作是没有专门上课的人标准，但能游，感觉还不错的（报班学的游得没我快）。后来自己还学会了不标准的仰泳，说不标准是因为我可以浮在水上，但需要边浮边游动，不能一直浮，我有一个和我一起自学，还有点胖的朋友就可以仰面浮很久。

根据我浅薄的经验，我感觉将游泳这件事当作需要达标的课程反而会阻碍学习。有考核但是时间又短，我觉得压力还是很大的。我感觉游泳是一个熟悉“水性”的事情，如果有很多训练标准和考核要求，我认为不容易让人进入状态。或者说，学习游泳就像一个“变野”的过程，顺从自己的身体适应水这个过程。我看以前游泳好手都是在水里玩出来的。这样的话，如果把游泳当作一件去探索，去放松，或者是在疲惫生活里一种冒险和“撒野”的方式，时不时都去玩一玩，可能会有别样的收获嘞。

因此，我觉得挫败感是教学体系带来的。因为符合教学标准而学会游泳是一种学习的方式，但十天还是太短啦。

**Reply by earfanfan** [reply]
嗯，你说得对，我真得就没体会到任何“玩水”的乐趣。挫败感很强也是因为课外看了很多视频讲一些新手常犯的错误，然而我真得游的时候还是犯了，理论知识相比实践过于丰富了……等我休息几天，再去接着练习，感受一下怎么“变野”，哈哈。

话又说回来，大学生还真是夜猫子党啊。

**Comment by hongtaoh** [comment]
我很好奇国内十节课多少钱。。

**Reply by hongtaoh** [reply]
我在加州学了四节课（只去了三节），花了我 200 块美元

**Reply by earfanfan** [reply]
我报名交钱是一对五的，就是一个教练一个班带五个学生，费用是1080元人民币。不过实际上课的是一个大班，那个费用实际上是700元。我很好奇你三节课学会了多少呢？

我觉得你在学习时的信念感应该会很强，应该不会像我那样难以体会到玩水的乐趣。我太胆小了……

**Reply by hongtaoh** [reply]
我报的一对四，但最后报名的只有我一个人，所以是一对一。总共四节课，一节1小时，但我最后一节没办法去，所以只去了三次。

三节课我只是去熟悉水性，基本上可以在水上浮起来，也学了如果仰着浮起来，其他的做不到了。

>我觉得你在学习时的信念感应该会很强，应该不会像我那样难以体会到玩水的乐趣

其实还好，我没有太多的期待，就只是尽量去试。这次学习让我觉得游泳挺有趣了，不像之前，一点都不会，就去游，就很受挫，觉得游泳不适合自己。

**Reply by earfanfan** [reply]
这边上课如果缺了某节课，后面还可以补课，毕竟是露天游泳池，天气很重要。仰着浮起来是怎么做到的？感觉后面如果你想继续学游泳的话，可以根据网上的课程自学了。

**Reply by hongtaoh** [reply]
这边不能补课 :(. 说到露天游泳池，我想到我在这里晒伤了。我印象中长这么大我几乎没有晒伤过。我的肩膀红得不像样子，一周之后才好，很痛，掉皮。仰着浮起来就是试着脸超天，试着漂浮在水上，我只能坚持几秒钟。

>感觉后面如果你想继续学游泳的话，可以根据网上的课程自学了。

我试试看！
